cg/cli/demultiplex/demux.py

```diff
@@ -24,54 +24,56 @@
     is_syncing_complete,
 )
 from cg.models.cg_config import CGConfig
-from cg.models.flow_cell.flow_cell import FlowCellDirectoryData
+from cg.models.illumina_run_directory_data.illumina_run_directory import (
+    IlluminaRunDirectory,
+)
 
 LOG = logging.getLogger(__name__)
 
 
 @click.command(name="all")
-@click.option("--flow-cells-directory", type=click.Path(exists=True, file_okay=False))
+@click.option("--sequencing-runs-directory", type=click.Path(exists=True, file_okay=False))

```

Comment:
I have not found the place where this path is defined though, maybe Im wrong, but better double check

Reply:
In servers the cli command is called as:

`cg demultiplex finish flow-cell all` There is no path given. To be quite honest. this function should not require a passable directory, since the directory is pulled from the config.

Reply:
I would say leave this change for another PR with only changes to the CLI commands